api/datadoghq/common/const.go

```diff
@@ -205,64 +124,64 @@ const (
-	SeccompSecurityVolumeName                        = "datadog-agent-security"
-	SeccompSecurityVolumePath                        = "/etc/config"
-	SeccompRootVolumeName                            = "seccomp-root"
-	SeccompRootVolumePath                            = "/host/var/lib/kubelet/seccomp"
-	SeccompRootPath                                  = "/var/lib/kubelet/seccomp"
-	SystemProbeSeccompKey                            = "system-probe-seccomp.json"
-	SystemProbeAgentSecurityConfigMapSuffixName      = "system-probe-seccomp"
-	SystemProbeSeccompProfileName                    = "system-probe"
+	LogDatadogVolumeName       = "logdatadog"
+	LogDatadogVolumePath       = "/var/log/datadog"
+	TmpVolumeName              = "tmp"
+	TmpVolumePath              = "/tmp"
+	CertificatesVolumeName     = "certificates"
+	CertificatesVolumePath     = "/etc/datadog-agent/certificates"
+	AuthVolumeName             = "datadog-agent-auth"
+	AuthVolumePath             = "/etc/datadog-agent/auth"
+	InstallInfoVolumeName      = "installinfo"
+	InstallInfoVolumeSubPath   = "install_info"
+	InstallInfoVolumePath      = "/etc/datadog-agent/install_info"
+	InstallInfoVolumeReadOnly  = true
+	PointerVolumeName          = "pointerdir"
+	PointerVolumePath          = "/opt/datadog-agent/run"
+	LogTempStoragePath         = "/var/lib/datadog-agent/logs"
+	PodLogVolumeName           = "logpodpath"
+	PodLogVolumePath           = "/var/log/pods"
+	ContainerLogVolumeName     = "logcontainerpath"
+	ContainerLogVolumePath     = "/var/lib/docker/containers"
+	SymlinkContainerVolumeName = "symlinkcontainerpath"
+	SymlinkContainerVolumePath = "/var/log/containers"
+	DogstatsdHostPortName      = "dogstatsdport"
+	DogstatsdHostPortHostPort  = 8125
+	DogstatsdSocketVolumeName  = "dsdsocket"
+	DogstatsdAPMSocketHostPath = "/var/run/datadog"
+	DogstatsdSocketLocalPath   = "/var/run/datadog"
+	DogstatsdSocketName        = "dsd.socket"
+	// SecurityAgentComplianceCustomConfigDirVolumeName = "customcompliancebenchmarks" // Not used
+	SecurityAgentComplianceConfigDirVolumeName   = "compliancedir"
+	SecurityAgentComplianceConfigDirVolumePath   = "/etc/datadog-agent/compliance.d"
+	SecurityAgentRuntimeCustomPoliciesVolumeName = "customruntimepolicies"
+	SecurityAgentRuntimeCustomPoliciesVolumePath = "/etc/datadog-agent-runtime-policies"
+	SecurityAgentRuntimePoliciesDirVolumeName    = "runtimepoliciesdir"
+	SecurityAgentRuntimePoliciesDirVolumePath    = "/etc/datadog-agent/runtime-security.d"
+	HostCriSocketPathPrefix                      = "/host"
+	CriSocketVolumeName                          = "runtimesocketdir"
+	RuntimeDirVolumePath                         = "/var/run"
+	KubeletAgentCAPath                           = "/var/run/host-kubelet-ca.crt"
+	KubeletCAVolumeName                          = "kubelet-ca"
+	APMHostPortName                              = "traceport"
+	// APMHostPortHostPort                              = 8126 // Not used
```

Comment:
i didn't remove these because i don't want to forget to check for consistency when moving these constants